website/docs/docs/dbt-versions/release-notes/04-Sept-2023/removing-prerelease-versions.md

```diff
@@ -0,0 +1,15 @@
+---
+title: "Removing old (prerelease) versions of dbt from dbt Cloud when (latest) is available"
+description: "Improving the version selection options by removing prerelease versions whenever the latest version is available."
+sidebar_label: "Removing old prerelease versions from dbt Cloud"
+tags: [Sept-2023, Versions]
+date: 2023-09-26
+sidebar_position: 07
+---
+
+Previously, when dbt released a new [version](/docs/dbt-versions/core#how-dbt-core-uses-semantic-versioning) in dbt Cloud, the older patch _prerelease_ version, in addition to the _latest_, remained as an option in the dropdown menu available in the **Environment settings**. Now, when the _latest_ version is released, the _prerelease_ version will be removed, and all customers remaining on it will be migrated seamlessly. There will be no interruptions to service when this migration occurs. 
```

Comment:
does it make sense to call out which page the **Environment settings** is on? 

```suggestion
Previously, when dbt Labs released a new [version](/docs/dbt-versions/core#how-dbt-core-uses-semantic-versioning) in dbt Cloud, the older patch _prerelease_ version and the _latest_ remained as options in the dropdown menu available in the **Environment settings**. Now, when the _latest_ version is released, the _prerelease_ version will be removed and all customers remaining on it will be migrated seamlessly. There will be no interruptions to service when this migration occurs. 
```